### ISSUE TITLE
Add 16:9 aspect ratio for n64 (and potentially all emulators)

### ISSUE THREAD

**Issue by klundberg** [rileytestut/Delta#88]
I was playing jet force Gemini for n64 on delta, and I noticed one of the game settings allowed me to play in widescreen. When I turned it on, the game changed how it rendered into a widescreen format but the viewport of the emulator (obviously) didn't change, keeping the area the game is rendered to 4:3. I'd love a way to support widescreen display on the n64 and potentially any emulator since other systems might also have reason to support this.

**Comment by klundberg** [comment]
I should note that this game would need a 'zoom' widescreen mode. I'm not sure if other n64 games out there would need a 'stretch' widescreen mode or not though, it would be nice to have both options.

**Comment by Ultra24** [comment]
Agreed, the feature really needs to be added and for some emulators like gba, the aspect ratio is almost 16:9 (it's aspect ratio for gba is 1.5:1 while standard widescreen is 1:77) so a stretched feature would be great as widescreen patches aren't easy for 2d games.

**Comment by WindowsNinetyTen** [comment]
That would be so cool, also for widescreen patched versions of games. I have a widescreen patched ROM of Mario Party 1 on this emulator, but it looks scrunched.
